routers/routes/goget.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package routes
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/util"
+	"github.com/unknwon/com"
```

Comment:
Just because you're already fiddling here, maybe we could replace `com.Expand` with `os.Expand`? 🙂

Reply:
it's doing something different from os.Expand.

It might be better to actually change this to use a go-template instead.

Reply:
Both are used to expand variables in strings. A template would also work.